app/otr/client/messageTypes.ts

```diff
@@ -0,0 +1,22 @@
+import { Meteor } from 'meteor/meteor';
+
+import { MessageTypes } from '../../ui-utils/client';
+import { otrSystemMessages } from '../lib/constants';
+
+Meteor.startup(() => {
+	MessageTypes.registerType({
+		id: otrSystemMessages.USER_JOINED_OTR,
+		system: true,
+		message: 'user_joined_otr',
```

Comment:
The **id** has same value as **message** this away here can be better.
 ```suggestion
		message: otrSystemMessages.USER_JOINED_OTR,
```

Reply:
Done!